### PR TITLE
{bio}[goolf/1.7.20] prokka v1.11

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+             'prokka-biocyc_to_fasta_db', 'prokka-clusters_to_hmm', 'prokka-hamap_to_hmm',
+             'prokka-uniprot_to_fasta_db', 'prokka-build_kingdom_dbs', 'prokka-genbank_to_fasta_db',
+             'prokka-make_tarball']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in bin_files] + ['binaries/linux/aragorn', 'db/cm/Bacteria', 'doc/ToDoList.txt'],
+    'dirs': ['bin', 'binaries', 'db',  'db/cm', 'db/genus', 'db/hmm', 'db/kingdom', 'doc'],
+}
+
+modloadmsg = "prokka scripts are located in \\$EBROOTPROKKA/bin; databases are located in \\$EBROOTPROKKA/db"
```
Comment (boegel) [inline easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb]: no need to escape the `$` anymore (cfr. #4035)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2028.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4cac1d3d46d31a72f069480ee431e129 for a full test report.

**Comment by boegel** [comment]
@pescobar Didn't seem to work for me, see failed test report?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
ca022.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5a599ad2aac50641f2e8b415038a36ea for a full test report.

**Comment by verdurin** [comment]
@pescobar please fix these style problems:
```
..../home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb:59:81: E261 at least two spaces before inline comment
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb:60:76: E261 at least two spaces before inline comment
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb:61:43: E261 at least two spaces before inline comment
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/p/prokka/prokka-1.11-goolf-1.7.20-Perl-5.22.2.eb:62:2: W299 trailing whitespace
```
and fix the URL in the header.

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**